# Definition of a region of interest (Primitive type with id)

# Optional
string id

# Required
geometry_msgs/PoseStamped pose

shape_msgs/SolidPrimitive primitive
